documentation/devnet.md

```diff
@@ -115,22 +115,53 @@ boostd -vv init \
   --max-staging-deals-bytes=2000000000
 ```
 
-11. Run boost
+11. Build the Web UI
+```
+make react
+```
+
+12. Edit config to set a fixed listen address
+
+Open `~/.boost/config.toml`
+
+Set the port in the `ListenAddresses` key to `50000` 
+```
+[Libp2p]
+  ListenAddresses = ["/ip4/0.0.0.0/tcp/50000", "/ip6/::/tcp/0"]
+```
+
+13. Run boost
 ```
 boostd -vv run
 ```
 
-12. Build the Web UI
+Note down the peer ID of the boost instance
```

Comment:
```suggestion
Note down the peer ID of the boost instance, it will be listed in the log output as `"maddr"` on a line that says `Boost libp2p node listening`.
```